README.md

```diff
@@ -192,19 +224,41 @@ This will exclude topics with names like "test", "test123", and "abc-testing".
 To override Jit's API endpoint, you can set the `JIT_API_ENDPOINT` environment variable. If the variable is not set, the
 default value will be used.
 
-### Settings Up Self-Hosted Runners
+## Settings Up Self-Hosted Runners
 
 To setup self-hosted runners, use the following command:
 
 You need to take the self hosted runners token from the Github Actions page of your repository.
 `https://github.com/<your-github-org-name>/jit/settings/actions/runners`
 
-#### Running on CentOS
+### Running on CentOS
```

Comment:
We need to add pre-requisit step to install git, and make  packages before running this command in both OSs

Reply:
it's in the readme in a lower part. moved it up..

that's from the readme:

## Prerequisites

- Python 3.x
- Git
- make

To make sure you have all you can run this command:

### Centos distro

```shell
sudo yum install -y git make && git clone https://github.com/jitsecurity/jit-customer-scripts.git && cd jit-customer-scripts
```

### Ubuntu distro

```shell
sudo apt install -y git make && git clone https://github.com/jitsecurity/jit-customer-scripts.git && cd jit-customer-scripts
```
### Mac

```shell
brew install git make && git clone https://github.com/jitsecurity/jit-customer-scripts.git && cd jit-customer-scripts
```